branding/core/core.jar/org/netbeans/core/startup/Bundle.properties

```diff
@@ -1,5 +1,5 @@
 #Updated by build script
-#Fri, 05 Jan 2018 10:31:22 -0500
+#Tue, 16 Jan 2018 14:02:15 -0500
```

Comment:
Please do a 

git checkout branding 

before committing, so that te branding file date changes are not included in your changes, thanks.